Docs: SigV4 What's New and AWS Elasticsearch documentation (#28506)
grafana/grafana: docs/sources/datasources/elasticsearch.md

      logMessageField: message
      logLevelField: fields.level
```

## Amazon Elasticsearch Service

> For AWS users using Amazon's Elasticsearch Service, it is possible to use Grafana's Elasticsearch Data Source to visualize your Elasticsearch data.

Reviewer: Not sure it makes sense to capitalize data source?
```suggestion
For AWS users using Amazon's Elasticsearch Service, it is possible to use Grafana's Elasticsearch data source to visualize your Elasticsearch data.
```

If you are using an IAM policy to control access to your Amazon Elasticsearch Service domain, you must use AWS Signature Version 4 (AWS SigV4) to sign all requests to that domain.
For more details on AWS SigV4, please see the [AWS documentation](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html).

### AWS Signature Version 4 Authentication

> Only available in Grafana v7.3+.

> In order to sign requests to your Amazon Elasticsearch Service domain, SigV4 can be enabled via the Grafana configuration file (it is disabled by default). 

Reviewer 2: ```suggestion
In order to sign requests to your Amazon Elasticsearch Service domain, SigV4 can be enabled in the Grafana configuration file. This feature is disabled by default. 
```

> In order to sign requests to your Amazon Elasticsearch Service domain, SigV4 can be enabled in the Grafana configuration file. This feature is disabled by default. 
> 
> ```ini
> [auth]
> sigv4_auth_enabled = true
> ```

Reviewer 3: Can we link to the configuration page instead?

```suggestion
In order to sign requests to your Amazon Elasticsearch Service domain, SigV4 can be enabled in the Grafana [configuration]({{< relref "../administration/configuration.md#sigv4_auth_enabled" >}}).
```

> Once AWS SigV4 is enabled, it can be configured on the Elasticsearch Data Source configuration page. The authentication options are documented [here](../datasources/cloudwatch.md#authentication).

Reviewer: ```suggestion
Once AWS SigV4 is enabled, it can be configured on the Elasticsearch data source configuration page. The authentication options are documented [here](../datasources/cloudwatch.md#authentication).
```

> Once AWS SigV4 is enabled, it can be configured on the Elasticsearch data source configuration page. The authentication options are documented [here](../datasources/cloudwatch.md#authentication).

Reviewer 4: ```suggestion
Once AWS SigV4 is enabled, it can be configured on the Elasticsearch data source configuration page. The authentication options are documented [here][here]({{<relref "../cloudwatch.md#authentication" >}})..
```

> Once AWS SigV4 is enabled, it can be configured on the Elasticsearch data source configuration page. The authentication options are documented [here]({{<relref "../cloudwatch.md#authentication" >}}).

Reviewer 2: ```suggestion
Once AWS SigV4 is enabled, it can be configured on the Elasticsearch data source configuration page. Refer to [Cloudwatch authentication]({{<relref "../cloudwatch.md#authentication" >}}) for more information about authentication options.
```

Reviewer 2: Per the [Grafana documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md#links-and-references), do not use blind links or vague link text such as "here.
